### ISSUE TITLE
Next steps for kw-detector

### ISSUE THREAD

**Issue by oliskir** [coastal-science/HALLO-models#1]
* [ ] Improve readme by adding more background and details
 * [ ] Expand annotations by 1 s in both directions to account for timestamp imprecision
 * [ ] Repeat model training and testing, using a smaller step size for sample time shifting (e.g. 0.6 s)

**Comment by fsfrazao** [comment]
If I may suggest another task, it would be good to have scripts that test a model in this repo as well, including steps to run it as a detector on the same test sets, compute and plot metrics.

I'd say this should be done before training a new model on a new training set.

**Comment by oliskir** [comment]
Good suggestion. I agree that this should be done as the next step, before any new training.
